test/functional/holds_controller_test.rb

```diff
@@ -24,56 +28,88 @@ class HoldsControllerTest < ActionController::TestCase
 
 
   test "should get new" do
-    get :new, params: { teacher_set_id: @hold1.teacher_set_id }
+    get :new, params: { teacher_set_id: @hold2.teacher_set_id }
     assert_response :success
   end
 
   test "should cancel hold" do
-    post :cancel, params: { id: @hold1.access_key }
+    post :cancel, params: { id: @hold2.access_key }
     assert_response :success
   end
 
   test "update teacher-set available_copies and cancel hold successfully" do
     sign_in @user
-    expected_ts_available_copies = 4
+    expected_ts_available_copies = 3
```

Comment:
I would just write `3` in line 59, not assign to a variable. Just my personal preference.